plugins/orchestrator/docs/rbac-policy.csv

```diff
@@ -0,0 +1,11 @@
+p, role:default/workflowViewer, orchestrator.workflowInstances.read, read, allow
```

Comment:
Do we need this file? We already have a sample in the `Permissions.md` file.

Reply:
easy to get started for developers.